repo/meta/schema/build-definition-schema.json

```diff
@@ -198,7 +197,6 @@
 
 
     "config": {
-      "$ref": "http://json-schema.org/draft-04/schema#"
```

Comment:
Is an empty `config` object allowed?

Reply:
Seems to be... It's not clear why the schema needed to be referenced again like this.